### ISSUE TITLE
UKTV not working

### ISSUE THREAD

**Issue by grapenose** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#1379]
Content no longer loads. 
Error Message-The selection triggered an unexpected error.
Have not tried uninstalling yet.

**Comment by snoopymail** [comment]
Quick question... Do you get audio with STV live?

**Comment by grapenose** [comment]
Was not aware that you can get STV live. Not listed as such.
Does STV play at all? No. 
Try's to load  but nothing happens.

**Comment by kevwag** [comment]
@grapenose did you actually read any of the bug report template before you deleted it? You've provided no information whatsoever to anyone in a position to investigate the issue and the bug report will just get ignored by the devs. 
I would suggest you edit your original post (don't post it down here - it won't get seen, it needs to be instantly obvious that you have provided the information you're supposed to when posting a bug report.)

On the subject of the problem itself, UKTV is in the process of rebranding to "U", the main URL has changed from _uktvplay.co.uk_ to _u.co.uk_ and the old URL now redirects to the new one in a web browser.
The browsing of catch-up content is still possible in the addon (I think this metadata lives on the similarly named _uktv.co.uk_ domain which currently appears unchanged) but attempting to play anything fails (after a considerable wait). The wording of the error message suggests a login/auth failure, probably down to the addon trying to authenticate on the old domain which no longer works.

It's worth noting that the live channels DO still work (although "W" channel has always been missing) but the four live channels are also being moved and rebranded.
The renaming of the channels will happen on 16th July 2024 and the rebrand will then be complete.
The Channels are renaming as follows:
"Dave" is becoming "U&Dave"
"Drama" is becoming "U&Drama"
"W" is becoming "U&W"
"Yesterday" is becoming "U&Yesterday"
(xml parsers are gonna love all those ampersands!)

The catch-up part of the site has already moved to _u.co.uk_ so that can probably be fixed in the addon as soon as someone's able to do it. The live channels will all be changing in a few days time so there's no point changing anything on the "Live" side until after the 16th.

**Comment by BugfinderPro135** [comment]
No, UKTV Play isn't working as is. Just installed the latest Beta 0.2.41.beta07 and it appears to start to play, but then just stops. As with most things Kodi, 1 step forward - 2 steps back....

**Comment by BugfinderPro135** [comment]
Na, still doesn't work...
